iocore/net/quic/QUICConfig.cc

```diff
@@ -462,6 +462,12 @@ QUICConfigParams::session_file() const
   return _session_file;
 }
 
+uint8_t
+QUICConfigParams::max_ack_only_packet() const
```

Comment:
Yes it should be. But it seems specs explicitly forbidden that. I'd like to obey the specs. Although it is out of draft-16 implementation.

Also I'd like to keep the `max_ack_only_packet` API but change the default value of the `_max_ack_only_packet` to 1 to avoid the future changes

Reply:
Why should it be configurable? For debug?

I'm ok with having it as configurable setting, but also want to make it clear what case we need to configure it so that we can configure it appropriately.

Reply:
Ok, forgot it. Let's remove it.

Reply:
Because it is not on draft-15 or draft-16. It may be changed due to any update in the future . The value of the `more than one packet` could be one or two or other value .

But I'm ok to update it with a bool value.